src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMapping.java

```diff
@@ -95,6 +99,30 @@ public static boolean shouldBeHearingsInWelshFlag() {
 
     public static List<HearingLocation> getHearingLocations(SscsCaseData caseData,
                                                             ReferenceDataServiceHolder referenceDataServiceHolder) {
+        String nextHearingVenueName = caseData.getAdjournCaseNextHearingVenue();
+
+        //TODO: remove flag
+        if (referenceDataServiceHolder.isAdjournmentFlagEnabled() && isNotEmpty(nextHearingVenueName)
+            && !HearingsChannelMapping.isPaperCase(caseData)) {
+            try {
```

Comment:
You shouldn't try catch here as we wouldn't want this to silently fail. I think you should add the exception to the method and it's parents.

Reply:
Done